### PR TITLE
[Docs] 202311 howto refactor review

### PR DESCRIPTION
**Pull request by databyjp**
Proposed changes to the `basic` and `similarity` files

### What's being changed:

- Add code examples fetching metadata
- Update `unvectorize input` examples to separate `nearText` and `nearImage` examples
- Remove groupby example from `basic`
- Syntax fixes / copyedit
    - TS example syntax fix
    - Update some `gql` based language to more generic language 
    - 
### Type of change:

- [x] **Documentation** updates (non-breaking change to fix/update documentation)

### How Has This Been Tested?

- [x] **Github action** – automated build completed without errors
- [x] **Local build** - the site works as expected when running `yarn start`

### CODE REVIEW

File: developers/weaviate/search/similarity.md [hunk 1]
```diff
@@ -14,29 +14,25 @@ import TSCode from '!!raw-loader!/_includes/code/howto/search.similarity.ts';
...
 
-## Search with unvectorized input
-
-If you use unvectorized inputs to search with, the collection must have a [vectorizer](../modules/retriever-vectorizer-modules/index.md) configured.
+## Search with text
 
-- For text object, use [`nearText`](../api/graphql/search-operators.md#neartext).
-- For image objects, use [`nearImage`](./image.md). 
-- For mixed text and images, use [CLIP](../modules/retriever-vectorizer-modules/multi2vec-clip.md).
+You can use use [`nearText`](../api/graphql/search-operators.md#neartext) to find objects with the nearest vector to an input.
```
Comment (sebawita) [inline developers/weaviate/search/similarity.md:35]: The example is just below.
Do we need a link to the reference page here? (this a question, not a directive)

File: developers/weaviate/search/similarity.md [hunk 2]
```diff
@@ -90,7 +86,48 @@ The output is like this:
...
 
-## Search for a similar object
+
+## Search with image
+
+import ImgSrchPyCode from '!!raw-loader!/_includes/code/howto/search.image.py';
+import ImgSrchPyCodeV3 from '!!raw-loader!/_includes/code/howto/search.image-v3.py';
+import ImgSrchTSCode from '!!raw-loader!/_includes/code/howto/search.image.ts';
+
+You can use [`nearImage`](../api/graphql/search-operators.md#nearimage) to find objects with the nearest vector to an input image. This example uses a base64 representation of an image.
```
Comment (sebawita) [inline developers/weaviate/search/similarity.md:96]: The example is just below.
Do we need a link to the reference page here? (this a question, not a directive)

Comment (databyjp) [reply developers/weaviate/search/similarity.md:96]: Same as above.

File: developers/weaviate/search/similarity.md (hunk 1 above)
Comment (sebawita) [reply developers/weaviate/search/similarity.md:35]: let's keep it then

**Pull request merged**